source/developers-guide/backend-components/basics/index.md

```diff
@@ -424,12 +424,12 @@ The property `$alias` will be the query alias used in every query with the root
 ## ExtJS Implementation
 Now, we go on to the proper usage of the backend components. First, we implement the following components in our directory `Views/backend`:
```

Comment:
I think the directory is `Resources/views/backend`, isn't it?
could you change that one, too?